src/test/java/by/innowise/monolithwebcalculator/service/CalculatorServiceTest.java

```diff
@@ -0,0 +1,74 @@
+package by.innowise.monolithwebcalculator.service;
+
+import by.innowise.monolithwebcalculator.domain.operation.Operation;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CalculatorServiceTest {
+
+    static CalculatorService calculatorService;
```

Comment:
Can you avoid using static `CalculatorService`?
I understand what you wanted to achieve but this is not the right way to do it. Move `calculatorService` initialization logic either to `@BeforEach` or take a look at `@TestInstance` annotation - https://www.baeldung.com/junit-testinstance-annotation